softserial_test/softserial_test.ino

```diff
@@ -0,0 +1,77 @@
+#include <ODriveArduino.h>
+#include <soft_uart.h>
```

Comment:
Super cool that you included this! Just for my understanding tho, we shouldn't need to be using soft serial ever cause of the teensy, correct?

Reply:
Same question. Is there a need for the software serial anymore?

Reply:
@mahajanrevant @agupta231 Correct. The Teensy has enough hardware serial ports to carry our needs. I left this in just to have as reference I suppose. Would you guys prefer if it was removed?

Reply:
I have no problem keeping it cause it's literally just a runner script. Since it works, it doesn't really hurt anyone.

Reply:
We can keep it but we probably should have a quick comment at the top that says with Teensy we don't need it. Just so whoever comes after us can get the context

Reply:
Comment added